securitas/controller/group.py

```diff
@@ -62,6 +63,11 @@ def group_add_member(ipa, groupname):
             return redirect(url_for('group', groupname=groupname))
 
         flash('You got it! %s has been added to %s.' % (username, groupname), 'success')
+
+        config.conf.setup_logging()
+        msg = schema.MemberSponsorV1()
```

Comment:
When trying to use my schema here, this fails with the message:

    TypeError: The class <class 'securitas.messaging.schema.MemberSponsorV1'> is not in the message registry, which indicates it is not in the current list of entry points for "fedora_messaging". Please check that the class has been added to your package's entry points.

This bit i added in the pyproject.toml was me trying to add it to the entry points, but it doenst seem to be wokring:

    [tool.poetry.plugins."fedora.messages"]
    "fas.group.member.sponsor.v1"="securitas.messaging.schema:MemberSponsorV1"